app/src/tests/components/contents/download/DownloadButton.test.tsx

```diff
@@ -0,0 +1,81 @@
+const mockDownload = jest.fn();
```

Comment:
maybe best to move under imports

Reply:
I think it has to go before the import so that the real download doesn't get pulled in by the imported method - will confirm this though,

Reply:
it wont.. when you use jest.mock it auto gets hoiste to top of file :smile:

Reply:
Yeah, that's what I thought it should do, but I'm sure it's been a problem in the past and only setting up mocks pre-imports fixed it. Will dig into that though, you're right, shouldn't need to do that.

Reply:
It was fine! Must have been having other issues when I did this before! 😆